src/linux/Packaging.Linux/build.sh

```diff
@@ -154,8 +154,9 @@ tar -czvf "$SYMTARBALL" * || exit 1
 popd
 
 # Build .deb
-INSTALL_TO="$DEBROOT/usr/bin/"
-mkdir -p "$DEBROOT/DEBIAN" "$INSTALL_TO" || exit 1
+INSTALL_TO="$DEBROOT/usr/local/bin/share/gcm-core/"
```

Comment:
```suggestion
INSTALL_TO="$DEBROOT/usr/local/share/gcm-core/"
```

..to match GCM Core on macOS.